src/Model/BaseModel.php

```diff
@@ -196,4 +196,20 @@ public function setDsResponse($Ds_Response)
     {
         $this->Ds_Response = $Ds_Response;
     }
+
```

Comment:
Add a comment explaining what the function does, also specify that returns an array. Thanks!